chunky/src/java/se/llbit/chunky/world/Material.java

```diff
@@ -60,6 +66,22 @@ public abstract class Material {
    */
   public float emittance = 0;
 
+  /**
+   * Offset to apply to the global emitter mapping exponent (the resulting value will be constrained to be >= 0).
+   */
+  public float emitterMappingOffset = 0;
+
+  /**
+   * Overrides the global emitter mapping type unless set to NONE.
+   */
+  public EmitterMappingType emitterMappingType = EmitterMappingType.NONE;
+
+  /**
+   * (x, y, z): The color to use for the REFERENCE_COLORS emitter mapping type.
+   * w: The range surrounding the specified color to apply full brightness.
+   */
+  public ArrayList<Vector4> emitterMappingReferenceColors = new ArrayList<>();
```

Comment:
We allocate lots of materials (one per block type, one per triangle in the bvh). Will this become an issue?

Reply:
Maybe I could set it to `null` and only initialize when needed?

Reply:
Sounds good

Reply:
Also, maybe set it to `null` again when it's not needed anymore